gradlew

```diff
@@ -0,0 +1,245 @@
+#!/bin/sh
```

Comment:
@CallMeCCLemon 
Did you created this file from scratch or we can look at the template file of it? It looks massive and I do not think I can create this file from scratch if I need to use grandlew